fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanAccrualWritePlatformServiceImpl.java

```diff
@@ -289,6 +289,8 @@ public Map<String, Object> deriveAccountingBridgeData(final LoanScheduleAccrualD
         accountingBridgeData.put("upfrontAccrualBasedAccountingEnabled", false);
         accountingBridgeData.put("periodicAccrualBasedAccountingEnabled", true);
         accountingBridgeData.put("isAccountTransfer", false);
+        accountingBridgeData.put("isChargeOff", false);
```

Comment:
Please mark this method to private to avoid anyone call it by mistake

Reply:
Fixed